### PR TITLE
config: add opae.cfg

### PR DESCRIPTION
**Pull request by tswhison**
Signed-off-by: Tim Whisonant <tim.whisonant@intel.com>

### CODE REVIEW

**Review by r-rojo — DISMISSED** [review]

File: opae.cfg [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+        ],
+        "rsu": []
+      }
+    },
+
+    "n6000": {
+      "enabled": true,
+      "platform": "Intel Acceleration Development Platform N6000",
+
+      "devices": [
```
Comment (anandaravuri) [inline opae.cfg:179]: Tom(Redhat) added  N6000 with out sub device id 
https://github.com/OPAE/opae-sdk/pull/2675

Beta1 release doesn't support  sub device id 

can you add to config file?

File: opae.cfg [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+        "rsu": []
+      }
+    },
+
+    "n5010": {
+      "enabled": true,
+      "platform": "Silicom FPGA SmartNIC N5010 Series",
+
+      "devices": [
+        { "name": "n5010_pf", "id": [ "0x1c2c", "0x1000", "0", "0" ] },
```
Comment (anandaravuri) [inline opae.cfg:131]: ["0x1c2c", "0x1000", "0", "0" ]    why "0"?  "*" may be correct ?
same as N3000 and D5005

File: opae.cfg (hunk 1 above)
Comment (tswhison) [reply opae.cfg:179]: It's actually there in the "ofs" section below.

File: opae.cfg (hunk 2 above)
Comment (tswhison) [reply opae.cfg:131]: It was set to "0" by Roger here: https://github.com/OPAE/opae-sdk/pull/2654/files

**Review by anandaravuri — APPROVED** [review]

**Pull request merged**